libnavigation-core/src/main/java/com/mapbox/navigation/core/routealternatives/NavigationRouteAlternativesObserver.kt

```diff
@@ -76,7 +76,8 @@ interface NavigationRouteAlternativesObserver {
      *                 // if the current route is onboard, and there's an offboard route available
      *                 // consider notifying the user that a more accurate route is available and whether they'd want to switch
      *                 // or force the switch like presented
-     *                 mapboxNavigation.setNavigationRoutes(offboardAlternatives)
+     *                 val legIndex = routeProgress.alternativeRoutesIndices[offboardAlternatives.first().id]?.legIndex ?: 0
+     *                 mapboxNavigation.setNavigationRoutes(offboardAlternatives, legIndex)
```

Comment:
offboard alternatives wasn't set yet, so they aren't present in the route progress, are they? 🤔

Reply:
Hmmmm, I think you're right...
Discussed with NN: it's going to be 0. Then I'll revert this doc and I can also make `RouteProgress#alternativeRoutesIndices` internal.